invenio_stats/ext.py

```diff
@@ -51,37 +47,53 @@ def __init__(self, app, events_entry_point_group):
         self.app = app
         self.exchange = app.config['STATS_MQ_EXCHANGE']
         self.suffix = app.config['STATS_INDICES_SUFFIX']
+        self.enabled_events = app.config['STATS_EVENTS']
         self.events_entry_point_group = events_entry_point_group
 
     @cached_property
     def _events_config(self):
         """Load events configuration."""
         # import iter_entry_points here so that it can be mocked in tests
-        from pkg_resources import iter_entry_points
         result = {}
-        for ep in iter_entry_points(group=self.events_entry_point_group):
-            entry = ep.load()
-            for cfg in ep.load()():
-                if cfg['event_type'] in result:
-                    raise DuplicateEventError(
-                        'Duplicate event {0} in entry point '
-                        '{1}'.format(cfg['event_type'], ep.name))
-
-            result[cfg['event_type']] = cfg
+        contrib_dev = False
```

Comment:
Added temporarily to register the 2 events coming from records-ui and files-rest from the contrib